app/views/install/codespaces/create-a-codespace.html

```diff
@@ -0,0 +1,84 @@
+{% set installEnvironment = "Codespaces" %}
+{% set current_step = '1' %}
+
+{% block pageTitle %}
+  Create a codespace - NHS prototype kit
+{% endblock %}
+
+{% extends 'install/codespace.html' %}
+
+{% block beforeContent %}
+  {{ breadcrumb({
+    items: [
+      {
+        href: "/",
+        text: "Home"
+      },
+      {
+    href: "/install/codespaces/",
+    text: "Install the kit in your web browser"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% block stepHeader %}
+  Create a Codespace
+{% endblock %}
+
+{% block page_content %}
+
+  <p class="nhsuk-lede-text">To use the kit within a Codespace you will first need to create one.</p>
+
+  <h2>1. Go to the prototype repository</h2>
+
+  <p>Go to the GitHub repository for the prototype you wish to use. For this guide we will be using the
+    <a href="https://github.com/nhsuk/nhsuk-prototype-kit">NHS prototype kit repository</a>.
+  </p>
+
+  <h3>If you do not have a prototype repository</h3>
+
+  <p>It is not possible right now to install the kit in a new Github repository unless you can install software on your computer.</p>
+
+  <p>You can try using Codespaces with the <a href="https://github.com/nhsuk/nhsuk-prototype-kit">NHS prototype kit repository</a>. You will not be able to save your progress.</p>
```

Comment:
This bit can now be updated to mention using the 'Use this template' feature on the NHS prototype kit repo on GitHub.

Reply:
I've done this now.